addons/project/static/src/js/tours/project.js

```diff
@@ -19,7 +19,7 @@ tour.register('project_tour', {
     position: 'bottom',
     edition: 'enterprise',
 }, {
-    trigger: '.o-kanban-button-new',
+    trigger: '.o-kanban-button-newa',
```

Comment:
Don't think pushing that was intended

Reply:
To test error logger i need to introduce errors ;)